rectorphp/rector-src
# platform agnostic asserts

- introduce PlatformAgnosticAssertions which handle cross platform issues
- apply PlatformAgnosticAssertions on FileRelocationResolverTest
- Update packages-tests/FileFormatter/ValueObject/NewLineTest.php
- apply PlatformAgnosticAssertions on MoveValueObjectsToValueObjectDirectoryRectorTest
- use PlatformAgnosticAssertions in AbstractTestCase
- Github Actions: run tests on windows-latest and ubuntu-latest
- fix assertSame() for non-string arguments
- added PlatformAgnosticAssertions::assertStringEqualsFile()
- eleminate false-return type from AddedFileWithContent::getRealPath()
- remove spurious PlatformAgnosticAssertions leftovers
- enable verbose phpunit to investigate 'Out of Memory' errors
- try a TimeAndMemoryTestListener
- remove verbose, not usefull and sloooooooooooooowww
- Revert "try a TimeAndMemoryTestListener"
- Ensure tests manually tear down props (#1)
- check for windows or linux line ending. \n is contained in both.
- support assertFileEquals() normalizing
- skip tests known to fail on windows
- skip another test
- fix ClassWithPublicPropertiesFactoryTest on windows
- simplify
- simplify and speedup newline normalizaion
- partly revert previous commit
- skip ReturnThisRemoveRectorTest on windows
- skip SensitiveHereNowDocRectorTest on windows
- removed useless `@throws`
- apply cs
